packages/core/src/components/search/SearchHistory/SearchHistory.tsx

```diff
@@ -1,13 +1,11 @@
-import { List as UIList, Button as UIButton } from '@faststore/ui'
-
-import Icon from 'src/components/ui/Icon'
-import Link from 'src/components/ui/Link'
+import {
+  SearchHistory as UISearchHistory,
+  SearchHistoryTerm as UISearchHistoryTerm,
+} from '@faststore/ui'
 import useSearchHistory from 'src/sdk/search/useSearchHistory'
 import useSearchInput from 'src/sdk/search/useSearchInput'
 
-import styles from '../search.module.scss'
-
-const SearchHistory = () => {
+const SearchHistory = ({ ...otherProps }) => {
```

Comment:
This is currently doing nothing apart from creating a copy of the props received by the component, so we don't need it. You can just receive `props` and spread it in line 20.

```suggestion
const SearchHistory = (props) => {
```